external/kafka/src/test/scala/org/apache/spark/streaming/kafka/KafkaRDDSuite.scala

```diff
@@ -40,43 +41,70 @@ class KafkaRDDSuite extends KafkaStreamSuiteBase with BeforeAndAfter {
     tearDownKafka()
   }
 
-  test("Kafka RDD") {
+  test("Kafka RDD basic usage") {
```

Comment:
nit: this is a KafkaRDDSuite ... so "KafkaRDD" does not need to added to every unit test name.:)